library/forms.inc

```diff
@@ -31,15 +31,17 @@ function getFormsByPid ($pid, $cols = "*")
 	return sqlQuery("select $cols from forms where pid ='$pid'");
 }
 
-function getFormByEncounter($pid,$encounter, $cols="form_id, form_name", $name="")
+function getFormByEncounter($attendant_id,$encounter, $cols="form_id, form_name", $name="")
 {
+	global $attendant_type;
  	$arraySqlBind = array();
-	$sql = "select $cols from forms where encounter = ? and pid=? ";
-	array_push($arraySqlBind,$encounter,$pid);
+	$sql = "select $cols from forms where encounter = ? ";
+	array_push($arraySqlBind,$encounter,$attendant_id);
```

Comment:
This will break if there is a $name. Need to add the $attendant_id after the below if block.